lib/Error/OAuth/InvalidClient.php

```diff
@@ -3,7 +3,11 @@
 namespace Stripe\Error\OAuth;
 
 /**
- * InvalidClient is raised when authentication fails.
+ * InvalidClient is raised when the client doesn't belong to you, or the API key mode (live or test)
+ * doesn't match the client mode. Or the stripe_user_id doesn't exist or isn't connected to your
+ * application.
```

Comment:
Not sure if this was copied from elsewhere, but the grammar is a little awkward. It reads "InvalidClient is raised if ..., or .... Or, [new sentence] ..."

It should probably be all one sentence with just a comma between the first and second possibilities and a single "or" between the second and third.